config/initializers/0_clowder-config.rb

```diff
@@ -1,19 +1,33 @@
 
+def build_endpoint_url ep
+  config = ClowderCommonRuby::Config.load
+  uri = if config.tls
+          # There is currently no tlsPort or anything similar in clowder-common-ruby
+          URI::HTTPS.build(host: ep&.hostname, port: ep&.tlsPort)
```

Comment:
[clowder-common-ruby](https://github.com/RedHatInsights/clowder-common-ruby) provides `.tls` which returns true or false so that we can check if TLS is being used or not but we need a field that provides the TLS port similar to how app-common-python provides `tlsPort`. This should be fine though as long as `config.tls` is false.